kpops/cli/pipeline_config.py

```diff
@@ -45,25 +45,31 @@ class PipelineConfig(BaseSettings):
         default=TopicNameConfig(),
         description="Configure the topic name variables you can use in the pipeline definition.",
     )
+    schema_registry_url: str | None = Field(
+        default=None,
+        example="http://localhost:8081",
+        env=f"{ENV_PREFIX}SCHEMA_REGISTRY_URL",
+        description="Address of the Schema Registry.",
+    )
     kafka_rest_host: str | None = Field(
         default=None,
         env=f"{ENV_PREFIX}REST_PROXY_HOST",
         example="http://localhost:8082",
-        description="Address to the rest proxy REST API.",
+        description="Address of the Kafka REST Proxy.",
     )
     kafka_connect_host: str | None = Field(
         default=None,
         env=f"{ENV_PREFIX}CONNECT_HOST",
         example="http://localhost:8083",
-        description="Address to the kafka connect REST API.",
+        description="# Address of Kafka Connect.",
     )
     timeout: int = Field(
         default=300,
         env=f"{ENV_PREFIX}TIMEOUT",
         description="The timeout in seconds that specifies when actions like deletion or deploy timeout.",
     )
 
-    create_namespace: bool = Field(default=False)
+    create_namespace: bool = Field(default=False)  # Move under helm_config?
```

Comment:
please move it :)

Reply:
I actually think that we should keep it there. HelmConfig contains only global flags, which `--create-namespace` isn't. 

Kinda unrelated, but it would be cool to have a `namespace` flag in `HelmConfig` instead of having to define it in `defaults.yaml`.